.changeset/smooth-crabs-protect.md

```diff
@@ -0,0 +1,6 @@
+---
+"@primer/css": patch
+---
+
+479-user-pics-misaligned-in-issues-assignment-view
```

Comment:
```suggestion
```

Not necessary here, the changeset code will compile it together and link to this pr example https://github.com/primer/css/releases/tag/v20.4.4